examples/cython/cython_examples/masked_log.pyx

```diff
@@ -0,0 +1,33 @@
+#!python
+# cython: embedsignature=True, binding=True
+
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+from libc.math cimport log
+import numpy as np
+cimport numpy as np
+
+def masked_log(x):
+    """x is a 1D numpy array"""
+    """returns -Inf for x <=0 and log(x) otherwise"""
```

Comment:
Probably should only have one doc string.

Reply:
Lalala not my fault all upstream. Updated to latest upstream.